apps/judicial-system/backend/src/app/modules/subpoena/subpoena.service.ts

```diff
@@ -31,11 +31,17 @@ export class SubpoenaService {
     @Inject(LOGGER_PROVIDER) private readonly logger: Logger,
   ) {}
 
-  async createSubpoena(defendant: Defendant): Promise<Subpoena> {
-    return await this.subpoenaModel.create({
-      defendantId: defendant.id,
-      caseId: defendant.caseId,
-    })
+  async createSubpoena(
+    defendant: Defendant,
+    transaction: Transaction,
+  ): Promise<Subpoena> {
+    return await this.subpoenaModel.create(
```

Comment:
You don't have to await here, so you may consider removing the await.

Reply:
Done ✔️